### PR TITLE
QA issues fix

### PR DESCRIPTION
**Pull request by Vinod-V3**

### CODE REVIEW

File: src/app/home/user-home/user-home.page.ts [hunk 1]
```diff
@@ -854,14 +854,15 @@ export class UserHomePage implements OnInit, OnDestroy, OnTabViewWillEnter {
     if (!event || !event.data || !event.data.length) {
       return;
     }
-    const title = event.data[0]['name'] === 'Project' ? 'FRMELEMENTS_MSG_YOU_MUST_JOIN_TO_PROJECT' :'FRMELEMENTS_MSG_YOU_MUST_JOIN_TO_OBSERVATIONS';
-    const meta = event.data[0]['name'] === 'Project' ? 'FRMELEMENTS_MSG_ONLY_REGISTERED_USERS_CAN_TAKE_PROJECT': 'FRMELEMENTS_MSG_ONLY_REGISTERED_USERS_CAN_TAKE_OBSERVATION';
+    let type = `${event.data[0]['name'].toLowerCase()}s`
```
Comment (vishwanath1004) [inline src/app/home/user-home/user-home.page.ts:857]: why this change?

File: src/app/manage-learn/project/attachment-listing/attachment-listing.page.html [hunk 2]
```diff
@@ -43,6 +43,6 @@ <h5 class="sb--card__title projects-labels"><span>{{'FRMELEMNTS_LBL_TASK_LEVEL'
     </div>
   </div>
   <div *ngIf="!attachments?.project?.attachments?.length && !attachments?.tasks?.length">
-    <app-no-data [message]="'FRMELEMENTS_NO_EVIDENCES_FOR_FILE'" [color]="'text-primary'"></app-no-data>
+    <app-no-data message="{{'FRMELEMENTS_NO_EVIDENCES_FOR_FILE' | translate:{'fileType':selectedTab} }}" [color]="'text-primary'"></app-no-data>
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/attachment-listing/attachment-listing.page.html:46]: what  is the issue for this change

File: src/app/manage-learn/project/project-details/project-details.component.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
         *ngIf="projectDetails?.status?.length &&  projectDetails?.status !== allStatusTypes.submitted" id="projectStatus">
         <p><strong>{{'YOUR_PROGRESS_LABEL' | translate}}</strong>
         </p>
-        <p>{{projectCompletionPercent?.completedTasks}} / {{projectCompletionPercent?.totalTasks}} {{'COMPLETED' |
+        <p>{{projectCompletionPercent?.completedTasks}} / {{projectCompletionPercent?.totalTasks}} {{'FRMELEMNTS_LBL_TASKS_COMPLETED' |
```
Comment (vishwanath1004) [inline src/app/manage-learn/project/project-details/project-details.component.html:16]: Why the literal changed

File: src/app/home/user-home/user-home.page.ts (hunk 1 above)
Comment (Vinod-V3) [reply src/app/home/user-home/user-home.page.ts:857]: When the program tile was added this condition was failing, as when a non loggedIn user clicks on the programs tile in the popup observation message was coming up

File: src/app/manage-learn/project/attachment-listing/attachment-listing.page.html (hunk 2 above)
Comment (Vinod-V3) [reply src/app/manage-learn/project/attachment-listing/attachment-listing.page.html:46]: When user selects each tab in the attachments list message should be shown dynamically based on the tab selection, ED-1371

File: src/app/manage-learn/project/project-details/project-details.component.html (hunk 3 above)
Comment (Vinod-V3) [reply src/app/manage-learn/project/project-details/project-details.component.html:16]: Based on this ticket ED-1430, the expected result should be like "1/1 tasks completed"
